### PR TITLE
Updated first examples to show model-free case.

### PR DESCRIPTION
**Pull request by MichaelClerx**
After discussion today, seems like we should start the first examples by showing you need an error/logpdf, then move on to show how PINTS lets you define one for time series

### CODE REVIEW

**Comment by MichaelClerx** [comment]
@ben18785 @chonlei @martinjrobins do you think this makes sense? It was a little unclear that you could use PINTS for any old function

**Review by k-shep — COMMENTED** [review]
On a related topic, I think the readme could also be updated to make it clear that PINTS can also be used for simpler problems. For instance, currently it starts by saying

'PINTS (Probabilistic Inference on Noisy Time-Series) is a framework for optimisation and Bayesian inference on ODE models of noisy time-series, such as arise in electrochemistry and cardiac electrophysiology.'

I think it would be good to make it clear that it can also be used with other models that aren't ODEs.

File: examples/sampling/first-example.ipynb
[the remark this answers is not in the document]
Comment (MichaelClerx) [reply examples/sampling/first-example.ipynb:1]: Thanks @k-shep ! I'll deal with the smaller points shortly.

With regards to the main introduction I think we should discuss in the meeting with the other @pints-team/devs ! We'll need to find a balance between the original USP (inference for time series) and wider applications - or decide to change PINTS' focus.

**Comment by MichaelClerx** [comment]
>  (I did actually think you couldn't use PINTS for 1D optimisation though - I'm pretty sure I encountered an error when I tried to use it for a 1D problem a while ago, and I thought someone else had told me that they'd also experienced this.)

I'm guessing you did what I did initially, and used a scalar parameter instead of a length-1 vector?

**Comment by k-shep** [comment]
> > (I did actually think you couldn't use PINTS for 1D optimisation though - I'm pretty sure I encountered an error when I tried to use it for a 1D problem a while ago, and I thought someone else had told me that they'd also experienced this.)
> 
> I'm guessing you did what I did initially, and used a scalar parameter instead of a length-1 vector?

I can't fully remember, but from looking back at my code I think you're probably correct. I do remember seeing an error raised by PINTS which said that it didn't support the 1D case. I think maybe @I-Bouros experienced something similar.

**Comment by I-Bouros** [comment]
> > > (I did actually think you couldn't use PINTS for 1D optimisation though - I'm pretty sure I encountered an error when I tried to use it for a 1D problem a while ago, and I thought someone else had told me that they'd also experienced this.)
> > 
> > 
> > I'm guessing you did what I did initially, and used a scalar parameter instead of a length-1 vector?
> 
> I can't fully remember, but from looking back at my code I think you're probably correct. I do remember seeing an error raised by PINTS which said that it didn't support the 1D case. I think maybe @I-Bouros experienced something similar.

That's true from my experience PINTS does not work with optimising or sampling single parameters.

**Comment by MichaelClerx** [comment]
OK Can you dig out a specific example? The notebooks in this PR show both a 1d optimisation and a 1d sampling problem, so it's definitely not impossible in general :D

**Comment by I-Bouros** [comment]
> OK Can you dig out a specific example? The notebooks in this PR show both a 1d optimisation and a 1d sampling problem, so it's definitely not impossible in general :D

I can describe the phenomenon, as it appears in my own work: I am working with an SEIR model for which I am trying to infer one parameter beta. I use my usual approach in which I write up a pints.LogPDF, pints.LogPrior and a pints.LogPosterior objects, and then when I try to run it (with 3 chains, so is not that issue), I get this error message:

Screenshot 2024-03-25 at 14 00 23

**Comment by MichaelClerx** [comment]
Oh, that's interesting! Seems to be coming directly from CMA-ES. We should add a notice to the CMA-ES optimiser to say it doesn't support 1-d optimisations. Alternatively could try and catch this error and replace it with a more informative one

**Comment by MichaelClerx** [comment]
- [x] The readme could be updated to make it clear that PINTS can also be used for simpler problems (than ODEs).
- [x] Pints --> PINTS
- [x] Explain ErrorMeasure vs LogPDF
- [x] More explanation of the output of PINTS and what it is doing
- [x] Optimisation: Printing found_parameters, found_value
- [x] Confusing sentence 'Anything that can be made to fit the ErrorMeasure interface...'
- [x] Confusing sentece: 'two main methods: One to tell the optimiser how many parameters the are, and one to run the simulations'
- [x] Optimisation: Show how a LogPDF can be used too 
- [x] Capitalise Markov, Gaussian
- [x] Sampling: "Thes" --> "These"
- [x] Note about 1D #1698

**Pull request merged**